README.md

```diff
@@ -11,18 +11,16 @@ All types are immutable and thread-safe.
 |
 [GitHub](https://github.com/typelevel/squants)
 |
-[User Forum](https://groups.google.com/forum/#!forum/squants)
-|
 [Wiki](https://github.com/typelevel/squants/wiki)
 
 [![Join the chat at https://gitter.im/garyKeorkunian/squants](https://badges.gitter.im/garyKeorkunian/squants.svg)](https://gitter.im/garyKeorkunian/squants?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
 ### Current Versions
-Current Release: **1.0.0**
-([API Docs](https://oss.sonatype.org/service/local/repositories/releases/archive/org/typelevel/squants_2.11/1.0.0/squants_2.11-1.0.0-javadoc.jar/!/index.html#squants.package))
+Current Release: **1.1.0**
+([API Docs](https://oss.sonatype.org/service/local/repositories/releases/archive/org/typelevel/squants_2.11/1.1.0/squants_2.11-1.1.0-javadoc.jar/!/index.html#squants.package))
 
 Development Build: **1.1.0-SNAPSHOT**
```

Comment:
The link to the API docs changed to 1.2.0-SNAPSHOT, so we should revert that then.